lib/form/validate.js

```diff
@@ -180,8 +180,8 @@ const buildValues = working => {
  *
  * @returns {true|FormInputValues} - Returns `true` if there are no validation or required issues, {FormInputValues} with error messages as values if there are
  */
-const required = (input, settings, validation) => {
-  if (input.target.required && (input.target.required === 'save' || input.target.required === 'publish') && input.target.value === '') {
+const valueCheck = (input, settings, validation) => {
+  if (input.target.hasOwnProperty('required') && (input.target.required === 'save' || input.target.required === 'publish') && input.target.value === '') {
```

Comment:
Will it? Isn't the the purpose of using `hasOwnProperty` here?
here is it working/not throwing:
http://codepen.io/scottnath/pen/gMPBoB?editors=1111